.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -27,19 +27,19 @@ jobs:
       - name: Install CDK
         run: npm install -g aws-cdk
       - name: Synthesize
-        run: cdk synthesize > exasolCloudwatchDashboard.yaml
+        run: cdk synthesize --json > exasolCloudwatchDashboard.json
```

Comment:
why json?

Reply:
They recommend json because its more stable. Parsing yaml seems to be a bit error prone. Json is more stable. See https://github.com/aws/aws-cdk/issues/2965 for details.